itez/users/api/serializers.py

```diff
@@ -12,3 +13,59 @@ class Meta:
         extra_kwargs = {
             "url": {"view_name": "api:user-detail", "lookup_field": "username"}
         }
+
+
+class ChangePasswordSerializer(serializers.Serializer):
+    """
+    Serializer for changing User password.
+    """
+    password = serializers.CharField(write_only=True, required=True, validators=[validate_password])
+    password2 = serializers.CharField(write_only=True, required=True)
+    old_password = serializers.CharField(write_only=True, required=True)
+
+    class Meta:
+        model = User
+        fields = ('old_password', 'password', 'password2')
+
+    def validate(self, attrs):
+        """
+        Validates if the fields password and password2 are equal.
+
+        Args:
+            attrs [collections.OrderedDict]: Contains data posted in the request body.
+
+        Raises:
+            [serializers.ValidationError]: An exception raise when password and password2 don't match.
+
+        Returns:
+            attrs [collections.OrderedDict]: Contains data posted in the request body.
+        """
+        if attrs['password'] != attrs['password2']:
+            raise serializers.ValidationError({"password": "Password fields didn't match."})
```

Comment:
just wondering here,are we showing the entered `password` in the error message too?

Reply:
Yes, we compare `password` and `password2(confirm password)` if they are equal.